src/main/java/nl/novi/Person.java

```diff
@@ -0,0 +1,109 @@
+package nl.novi;
+
+import com.sun.source.tree.ReturnTree;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+
+public class Person {
+
+  private String name;
+  private String middleName;
+  private String lastName;
+  private String sex;
+  private int age;
+  private Person mother;
+  private Person father;
+  private Person partner;
+  private List<Person> siblings = new ArrayList<>();
+  private List<Person> children = new ArrayList<>();
+  private List<Pet> pets = new ArrayList<>();
+

```

Comment:
Voor de pets List zou je de new ArreyList<>() ook aan kunnen maken in de addPet methode.